docs/streaming_endpoints.md

```diff
@@ -0,0 +1,50 @@
+# Streaming API (preview) {#ovms_docs_streaming_endpoints}
+
+## Introduction
+OpenVINO Model Server implements gRPC KServe extension which adds separate RPC for [bidirectional streaming](https://grpc.io/docs/what-is-grpc/core-concepts/#bidirectional-streaming-rpc) use cases. It means that besides unary RPC, where client sends a single request and gets back a single response, client is able to initiate connection and send/receive messages in any number and order using `ModelStreamInfer` procedure:
+
+```
+service GRPCInferenceService
+{
+  ...
+  // Standard
+  rpc ModelInfer(ModelInferRequest) returns (ModelInferResponse) {}
+  // Extension
+  rpc ModelStreamInfer(stream ModelInferRequest) returns (stream ModelStreamInferResponse) {}
+  ...
+}
+```
+
+This becomes very useful for endpoints serving [MediaPipe Graphs](./mediapipe.md). In unary inference RPC each request initializes new MediaPipe graph, inputs are deserialized into MediaPipe packets and pushed into the graph. After the graph is done, it is deleted.
+However, in streaming inference RPC MediaPipe graph is created only once per connection, and is reused by subsequent requests to the same gRPC stream. This removes graph initialization overhead and increases its overall throughput.
```

Comment:
```suggestion
However, in streaming inference RPC, MediaPipe graph is created only once per the client session and is reused by subsequent requests to the same gRPC stream. This enables handling stateful graphs where results depends on the previous state. It also adds support graphs with Mediapipe `source calculators` and enables special cases when the request timestamps can be defined by the client. Finally, it removes graph initialization overhead and increases its overall throughput.
```

Reply:
Timestamp and statefulness is explained below. I think it's not worth mentioning about everything in one paragraph @dtrawins